tensorboard/backend/empty_path_redirect_test.py

```diff
@@ -38,7 +38,9 @@ def setUp(self):
         app = empty_path_redirect.EmptyPathRedirectMiddleware(app)
         app = self._lax_strip_foo_middleware(app)
         self.app = app
-        self.server = werkzeug_test.Client(self.app, werkzeug.BaseResponse)
+        self.server = werkzeug_test.Client(
+            self.app, werkzeug.wrappers.BaseResponse
```

Comment:
OK: Confirmed that you don’t need to separately import `wrappers` and
that it’s available as `werkzeug.wrappers` by default. LGTM.